service/awsconfig/v10/resource/loadbalancer/delete.go

```diff
@@ -7,19 +7,27 @@ import (
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/elb"
 	"github.com/giantswarm/microerror"
-	"github.com/giantswarm/operatorkit/framework"
+
+	"github.com/giantswarm/aws-operator/service/awsconfig/v10/key"
 )
 
-func (r *Resource) ApplyDeleteChange(ctx context.Context, obj, deleteChange interface{}) error {
-	deleteInput, err := toLoadBalancerState(deleteChange)
+// EnsureDeleted ensures that any ELBs from Kubernetes LoadBalancer services
+// are deleted. This is needed because the use the VPC public subnet.
+func (r *Resource) EnsuredDeleted(ctx context.Context, obj interface{}) error {
+	customObject, err := key.ToCustomObject(obj)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	lbState, err := r.clusterLoadBalancers(customObject)
```

Comment:
ApplyDeleteChange becomes EnsureDeleted and we get the cluster load balancers.

Reply:
That's nice change. Also loads of javascript style tests go away :)